Gemfile.lock

```diff
@@ -51,7 +51,7 @@ GEM
       public_suffix (>= 2.0.2, < 4.0)
     arel (8.0.0)
     ast (2.3.0)
-    autoprefixer-rails (7.2.1)
```

Comment:
Not sure why all these gems are being updated.

Reply:
@pmanrubia Would be happy to pair with you on this